index.html

```diff
@@ -24,43 +28,237 @@
+              <li><a>HTML</a></li>
+              <li><a>Bootstrap</a></li>
+              <li><a>Ruby</a></li>
+            </ul>
+          </div>
+          <button class="button-project-card flex-row">See project</button>
+      </article>
+    </section>
+    <section class="cards">
+      <article class="single-card flex-row">
+        <div>
+          <h4 class="card-title">Professional Art Printing Data</h4>
+        </div> 
+          <div class="infography-card">
+            <p>
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+          </div>
+          <div class="list-technologies-card">
+            <ul class="list-technologies-card-ul flex-row">
+              <li><a>HTML</a></li>
+              <li><a>Bootstrap</a></li>
+              <li><a>Ruby</a></li>
+            </ul>
+          </div>
+          <button class="button-project-card flex-row">See project</button>
+      </article>
+    </section>
+    <!--About me section-->
+    <section class="about-me flex-column">
+      <div class="about-top">
+        <h2 class="title-about">About me</h2>
+        <div class="infography-about">
+          <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="button-cont-resume">
+          <button class="button-resume flex-row">Get my resume</button>
+        </div>
+    </div>
+    <div>
+      <figure class="vector flex-row">
+        <img src="./img/Vector 2.png" alt="vector">
+      </figure>
+    </div>
+      <div class="about-bottom flex-column">
+        <div class="ab-bt-blocks flex-column">
+          <ul>
+            <h4 class="ab-titles languages"> Languages</h4>
+            <li class="about-bullet">Javascript</li>
+            <li class="about-bullet">Ruby</li>
+            <li class="about-bullet">HTML</li>
+            <li class="about-bullet">CSS</li>
+          </ul>
+        </div>
+        <div class="ab-bt-blocks">
+          <ul>
+            <h4 class="ab-titles frameworks"> Frameworks</h4>
+            <li class="about-bullet">Reac.js</li>
+            <li class="about-bullet">Ruby on Rails</li>
+            <li class="about-bullet">Rspec</li>
+            <li class="about-bullet">Capybara</li>
+          </ul>
+        </div>
+        <div class="ab-bt-blocks">
+          <ul>
+            <h4 class="ab-titles skills"> Skills</h4>
+            <li class="about-bullet">Database Management</li>
+            <li class="about-bullet">Version Control</li>
+            <li class="about-bullet">CLI</li>
+            <li class="about-bullet">API Design</li>
+            <li class="about-bullet">Web Development</li>
```

Comment:
- [ ] Please make sure that you adjust the spacing interval between the listed skills of this section to match the Figma design 

  ### YOUR DESIGN
  ![image](https://github.com/Rebelzob/Portfolio/assets/92303891/3d29cd2b-8c33-45dd-a0ab-28cc214413e8)
  
  ### FIGMA DESIGN
  ![image](https://github.com/Rebelzob/Portfolio/assets/92303891/54abd7dc-984c-4ade-bd2b-6002544b3e5d)

Reply:
@Owusu-Desmond thanks for your review!!